cli/src/command/wallet.rs

```diff
@@ -12,14 +12,26 @@ use log::LevelFilter;
 
 use crate::{error::Error, helper::get_password, println_log_info};
 
+const DEFAULT_NODE_URL: &str = "https://api.testnet.shimmer.network";
+const DEFAULT_WALLET_DATABASE_PATH: &str = "./stardust-cli-wallet-db";
+const DEFAULT_STRONGHOLD_SNAPSHOT_PATH: &str = "./stardust-cli-wallet.stronghold";
```

Comment:
This is a breaking change, do we really want that?

Reply:
I don't see it. :smile:

Reply:
Those strings are exactly the same defaults. This PR is breaking due to something else though.